insights/specs.py

```diff
@@ -203,6 +203,36 @@ def docker_container_ids(broker):
 ipv4_neigh = sf.simple_command("/sbin/ip -4 neighbor show nud all", name="ipv4_neigh")
 ipv6_neigh = sf.simple_command("/sbin/ip -6 neighbor show nud all", name="ipv6_neigh")
 iscsiadm_m_session = sf.simple_command("/usr/sbin/iscsiadm -m session", name="iscsiadm_m_session")
+ps_auxww = sf.first_of([
```

Comment:
I'd _really strongly_ recommend not reordering the ps parser specs just to be above the jboss_standalone_main_config_files spec.  I can see why you're doing it, but we're trying to keep the parser specs in alphabetical order for ease of reference.

What I'd recommend doing here is putting the more complex data sources at the bottom of the specs file, in a specific place so that we know to look there.  Then leave a comment where the 'jboss_standalone_main_config_files' spec would normally be defined to say that it's down below.

However, I think @csams was talking about a better way to include data sources in parsers, so that we didn't have this kind of complex dep

Reply:
@PaulWay Agree and I add my datasource to the end of the spec for now before Chris give us a better way.